README.md

```diff
@@ -97,6 +97,41 @@ arbitrary LLVM version.
          LLVM_SYSPATH=$LLVM_BUILD_DIR \
          pip install -e python
 
+# Tips for building
+
+- Set `TRITON_BUILD_WITH_CLANG_LLD=true` as an environment variable to use clang
+  and lld.  lld in particular results in faster builds.
```

Comment:
We should probably not keep that unless we have a fix for it?

Reply:
I think we should switch to clang + lld by default